### ISSUE TITLE
BG - Floats appear to use way more decimal places than necessary

### ISSUE THREAD

**Issue by j-conrad** [mozilla/hubs#6343]
**Description**
When dealing with `Float` values, the output shows 17 decimal places. I might expect this when working with values such as `Position`, however, even declaring a variable and setting it to `0.1` produces this output in the console log: `0.10000000149011612`

This is difficult to read and could possibly be contributing to calculation errors, especially when using comparison nodes like `greater than`.

**To Reproduce**
1. Open the attach `.blend` file
2. Note the graph that increases `MyInt` by `0.1` each time the monkey is clicked
3. Export as .glb and run in Hubs
4. Open the console log and watch the values when clicking the monkey

**Expected behavior**
If a user explicitly sets a float to a number, I would expect it to use and display that exact number.

**Screenshots**
![image](https://github.com/mozilla/hubs/assets/31968496/51215a1a-1f22-4791-bf6f-123161992c50)

**Hardware** 
 - Device: Desktop
 - OS: Windows
 - Browser: Chrome

[FloatingPointDecimals.zip](https://github.com/mozilla/hubs/files/13048443/FloatingPointDecimals.zip)

**Comment by Exairnous** [comment]
TLDR; that's just the way floats are.  Other data types could possibly be used, but they come with their own drawbacks.  If we aren't already, switching to 64-bit floats might be a good idea though.
https://www.youtube.com/watch?v=PZRI1IfStY0
https://blog.demofox.org/2017/11/21/floating-point-precision/

**Comment by Exairnous** [comment]
Here's an article that explains how to handle the inaccuracy when comparing whether two floating point numbers are equal: https://www.geeksforgeeks.org/problem-in-comparing-floating-point-numbers-and-how-to-compare-them-correctly/
And I think for greater than/less than comparisons you'd use rounding, or just pad the comparison enough to account for the imprecision.
Maybe the comparison nodes should account for this automatically, if they don't already.
